index.html

```diff
@@ -178,5 +178,58 @@ <h4 class="skill-title">Skills</h4>
       </div>
     </div>
   </section>
+  <section class="section-contact-form">
+    <div class="contact-form-container grid gen-margin">
+      <h3 class="contact-form-primary">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+      <form action="" class="contact-form">
+        <input type="text" class="form-input" placeholder="Full name">
```

Comment:
- [ ] Name input should have  `maxlength="30"` attribute. This is so as specified in the project requirements, to add validation to your form 👍 
- All your input fields should also  have the `required` attribute to activate the validation on submit.